### ISSUE TITLE
GPU setting

### ISSUE THREAD

**Issue by Scott-che** [uncbiag/SimpleClick#13]
Can I train your model on single GPU?
If the answer is 'Yes', what hyperparameter that I need to revise

**Comment by Scott-che** [comment]
Thanks for your help. The last two day I had tried several times train base_448_sbd, and I found that the training loss went to rounding 0.31 after two or three epochs. But the training loss didn't keep going down after two or three epochs.
Expect the refactoring version from you.
